projects/fuzzing-ethereum-network.md

```diff
@@ -0,0 +1,107 @@
+# Fuzzing the Ethereum network
+
+Create fuzzers for the ethereum network in order to find potential vulnerabilities .
+
+## Motivation
+
+In order to ensure the good running of Ethereum, it is imperative that is remains highly secure. As a large blockchain, Ethereum represents a target for potential hackers. 
+
+The Ethereum network and the differents protocols of the devp2p, plays an essential role in enabling communication between nodes, an essential element for the operation of the protocol. 
+
+Vulnerabilities or errors in this network could cause significant problems. It is therefore crucial to guarantee the maximum security of the network and exchanges between nodes.
+
+## Project description
+
+To increase security, I propose the creation of fuzzers, with the aim of potentially spotting vulnerabilities and bugs present on the Ethereum network and clients implementing the devp2p.
+
+The project was originally proposed by [Fredrik](https://github.com/fredriksvantes).
+
+The fuzzers will allow arbitrary/specific data to be sent to the various ethereum nodes through its network to cause unexpected behavior such as crashes, memory leaks and others...
```

Comment:
I would rephrase this to be a bit more specific. Something along `Fuzzers can build and send random or specified payloads over devp2p. Goal is to find potential triggers for DoS vectors, crashes..`

Reply:
Important to explain you are not gonna spam random mainnet nodes but creating a testing environment